### PR TITLE
Learnt phrase

### PR DESCRIPTION
**Pull request by ravinalamada**
Hello team,

This is a component that displays the phrases that a user has learned and stored that data locally by using thunk. Can you review it  at any time you have a chance for it please?

Thanks in advanced,

### CODE REVIEW

File: src/screens/Learning.js [hunk 1]
```diff
@@ -1,5 +1,6 @@
 import React, {useState, useEffect, useCallback} from 'react';
 import {action} from '@storybook/addon-actions';
+// import AsyncStorage from '@react-native-async-storage/async-storage';
```
Comment (andrzej-onja) [inline src/screens/Learning.js]: remove unused code

File: src/screens/Home.js [hunk 2]
```diff
@@ -124,12 +137,19 @@ export default ({
             <SectionHeading text="Learnt phrases:" />
           </View>
           <List
-            data={[{id: 2, name: '10 words and phrases'}]}
+            data={[
+              {
+                id: 2,
```
Comment (andrzej-onja) [inline src/screens/Home.js]: instead of 2 as Id use something like ###learnt-phrases###

File: src/screens/Learning.js [hunk 3]
```diff
@@ -133,7 +143,11 @@ export default ({
           </View>
           <View style={styles.heading}>
             <SectionHeading text="Category: " />
-            <Text>{currentCategoryName}</Text>
+            <Text>{`${
+              learntPhrasesCat
```
Comment (andrzej-onja) [inline src/screens/Learning.js]: as discussed we dont need 'learntPhrasesCat' in redux state try to set current category when clicking on the main screen. Set it to id ###learnt-phrases###
Then pass the existing prop current cat id from state here and do the condition logic based on that.

File: src/screens/Learning.js [hunk 4]
```diff
@@ -90,10 +96,15 @@ export default ({
...
     setCurrentPhrase(newPhrase);
-
     setAnswerOptionsCallback(originalAll, newPhrase);
   };
 
+  const learntCategory = categories.find(cat =>
+    cat.phrasesIds.includes(currentPhrase?.id),
+  );
+
+  const learntCatName = learntCategory?.name.en;
```
Comment (lightme-fan) [inline src/screens/Learning.js:106]: This is always going to stay in English. Perhaps, you use .[LANGUAGE_NAMES.EN].

**Review by lightme-fan — CHANGES_REQUESTED** [review]
When you display the lerntCatName, use the LANGUAGE_NAMES.EN].

**Review by Tiaraha-funny — APPROVED** [review]

**Review by lightme-fan — APPROVED** [review]

**Review by andrzej-onja — APPROVED** [review]

**Pull request merged**